notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1882 @@
+{
```

Comment:
<p>"...so here you will just use <code>get_time_series_dense_encoder_forecasting_pipeline_and_parameters</code> as the example..."</p><p>Our writing guidelines (https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template_writing_guidelines.md) discourage future tense (you will...). Please rewrite using present tense (...so here, you just use...). There are many cases of this in the notebook, please search and correct all instances. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Fixed.</p>